src/coreclr/vm/typestring.cpp

```diff
@@ -760,8 +790,47 @@ void TypeString::AppendType(TypeNameBuilder& tnb, TypeHandle ty, Instantiation t
     // ...or function pointer
     else if (ty.IsFnPtrType())
     {
-        // Don't attempt to format this currently, it may trigger GC due to fixups.
-        tnb.AddName(W("(fnptr)"));
+        // Currently function pointers return NULL for FullName and AssemblyQualifiedName and "*()" for Name.
+        // We need a grammar update in order to support parsing (see https://docs.microsoft.com/en-us/dotnet/framework/reflection-and-codedom/specifying-fully-qualified-type-names)
```

Comment:
Please remove the `en-us/` from all URLs and let the user's browser dictate the locale.